mopidy_spotify/playback.py

```diff
@@ -78,6 +83,7 @@ def change_track(self, track):
             sp_track.load(self._timeout)
             self.backend._session.player.load(sp_track)
             self.backend._session.player.play()
+            _held_buffer = None
```

Comment:
@jodal probably knows the answer to this better than me, but when would the session start being able to fill this value async? I.e. should we reset it before calling load/play, or does it depend on the current state of the session?

Could protecting this global with a lock be reasonable, or are we just asking for deadlocks if we aren't careful?

Reply:
It should be fine if placed before. My mistake I didn't think it could happen. I'll correct this.